Lab3/binary_search_tree.cpp

```diff
@@ -54,47 +54,64 @@ unsigned int BinarySearchTree::height() const {
 }
 
 // PURPOSE: Prints the contents of the tree; format not specified
-// TODO
+// TODO @Sammy
 void BinarySearchTree::print() const {
 }
 
 // PURPOSE: Returns true if a node with the value val exists in the tree
 // otherwise, returns false
-bool BinarySearchTree::exists( BinarySearchTree::TaskItem val ) const {
-    return false;
+// TODO @Sammy
+
+
+bool BinarySearchTree::exists(BinarySearchTree::TaskItem val) const {
+    if (root == NULL) {
+        cerr << "Empty tree\n";
+        return false;
+    } else if (size == 1) {
+        return val == *root ? true : false;
```

Comment:
I don't think we need the ` ? true : false; `

Reply:
if its of size one and it equals the root then return true, else return false?

Reply:
oh like it's not important just `return val == * root;` means the same thing as `return val == *root ? true : false`

Reply:
okay ya thats cleaner ill do that

Reply:
fixed in latest commit